pkg/compactor/compactor.go

```diff
@@ -302,6 +303,10 @@ func newCompactor(
 			Name: "cortex_compactor_tenants_processing_failed",
 			Help: "Number of tenants failed processing during the current compaction run. Reset to 0 when compactor is idle.",
 		}),
+		compactionRunInterval: promauto.With(registerer).NewGauge(prometheus.GaugeOpts{
+			Name: "cortex_compactor_run_interval_seconds",
```

Comment:
Maybe `cortex_compactor_compaction_interval_seconds` to keep it consistent with the config option name `compaction_interval`?